src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1123,16 +1124,18 @@ private void SetGeometryFrozen(HashSet<NodeModel> gathered)
 
         private void SetSelection(IEnumerable<Element3D> items, bool isSelected)
         {
-         
+            var postEffect = isSelected ? borderPostEffectString : null;
             foreach (var element in items)
             {
                 AttachedProperties.SetShowSelected(element, isSelected);
+                (element as GeometryModel3D).PostEffects = postEffect;
```

Comment:
How does this relate to the `ShaderPassDescription` for `DefaultPassNames.EffectOutlineP1`?